### ISSUE TITLE
real-time-face-mask-detector-with-tensorflow-keras-and-opencv

### ISSUE THREAD

**Issue by budisfg** [budisfg/article#47]
https://towardsdatascience.com/real-time-face-mask-detector-with-tensorflow-keras-and-opencv-38b552660b64?gi=51e04ac9d336
